src/Abp.Zero.Common/Authorization/Roles/AbpRolePermissionCacheItemInvalidator.cs

```diff
@@ -29,6 +33,16 @@ public void HandleEvent(EntityChangedEventData<OrganizationUnitRole> eventData)
         {
             var cacheKey = eventData.Entity.RoleId + "@" + (eventData.Entity.TenantId ?? 0);
             _cacheManager.GetRolePermissionCache().Remove(cacheKey);
+
+            //get all users in organization unit
+            var users = _userOrganizationUnitRepository.GetAllList(x =>
```

Comment:
should we consider adding 
```csharp
public void HandleEvent(EntityChangedEventData<OrganizationUnitRole> eventData)
{
   // more code ...
}
```

into `AbpUserPermissionCacheItemInvalidator` instead?

So that `_cacheManager.GetUserPermissionCache().Remove(cacheKey);` are grouped in the same handler

Reply:
Yes, it seems better. I have committed the requested changes. Thanks @ryancyq.